magma/platform/platform_glfw.c

```
@@ -0,0 +1,100 @@
+    mg_input_process_mouse_wheel((int8_t)yoffset);
+}
+
+mg_platform *mg_platform_initialize(mg_platform_init_info *init_info)
+{
+    mg_glfw_platform *platform = (mg_glfw_platform*)malloc(sizeof(mg_glfw_platform));
+
+    platform->window_width = init_info->width;
+    platform->window_height = init_info->height;
+
+    if (!glfwInit())
+        return NULL;
```

Cppcheck (2.10-2)

[error] Memory leak

(memleak)